test/fuzz/test_runner.py

```diff
@@ -100,6 +108,11 @@ def main():
 
     logging.info("{} of {} detected fuzz target(s) selected: {}".format(len(test_list_selection), len(test_list_all), " ".join(test_list_selection)))
 
+    if args.generate:
```

Comment:
Any reason to not reuse the fuzz_pool below? Benefit is also that it will check for libFuzzer and abort early if not found

Reply:
The main reason was to avoid the seedless check just below

Reply:
The check is just a debug print, no? If it does any harm, you could disable it when `args.generate`.

Though, I like the libFuzzer check to run before any libFuzzer specific args are passed to a subprocess.

Reply:
Ok, will change